tests/utils/testutils.py

```diff
@@ -1046,11 +1024,11 @@ def _setup_func_app(app_root):
         with open(host_json, 'w') as f:
             f.write(HOST_JSON_TEMPLATE)
 
-    if not os.path.isfile(extensions_csproj_file):
+    if not os.path.isfile(extensions_csproj_file) and not is_unit_test:
```

Comment:
so for unit tests, we never need extensions installed, only http tests?

Reply:
Right, we dont actually need extensions for unit tests,.